[CI] ClientYamlTestSuiteIT test {yaml=search.aggregation/210_top_hits_nested_metric/top_hits aggregation with sequence numbers} failing

Looks to be related to #84458 which hasn't been backported (but should be?)

**Build scan:**
https://gradle-enterprise.elastic.co/s/4wl6lwj7ss3gq/tests/:rest-api-spec:yamlRestTestV7CompatTest/org.elasticsearch.test.rest.ClientYamlTestSuiteIT/test%20%7Byaml=search.aggregation%2F210_top_hits_nested_metric%2Ftop_hits%20aggregation%20with%20sequence%20numbers%7D

**Reproduction line:**
`./gradlew ':rest-api-spec:yamlRestTestV7CompatTest' --tests "org.elasticsearch.test.rest.ClientYamlTestSuiteIT" -Dtests.method="test {yaml=search.aggregation/210_top_hits_nested_metric/top_hits aggregation with sequence numbers}" -Dtests.seed=C4F70F390EC7D2D0 -Dtests.locale=ja -Dtests.timezone=America/St_Barthelemy -Druntime.java=17`

**Applicable branches:**
8.1

**Reproduces locally?:**
Yes

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.test.rest.ClientYamlTestSuiteIT&tests.test=test%20%7Byaml%3Dsearch.aggregation/210_top_hits_nested_metric/top_hits%20aggregation%20with%20sequence%20numbers%7D

**Failure excerpt:**
```
java.lang.AssertionError: Failure at [search.aggregation/210_top_hits_nested_metric:168]: expected [2xx] status code but api [search] returned [400 Bad Request] [{"error":{"root_cause":[{"type":"query_shard_exception","reason":"it is mandatory to set the [nested] context on the nested sort field: [users.last.keyword].","index_uuid":"wrg-FT1JRPSAIPWt5qzYTw","index":"my-index","stack_trace":"[my-index/wrg-FT1JRPSAIPWt5qzYTw] org.elasticsearch.index.query.QueryShardException: it is mandatory to set the [nested] context on the nested sort field: [users.last.keyword].\n\tat org.elasticsearch.search.sort.FieldSortBuilder.validateMissingNestedPath(FieldSortBuilder.java:665)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.nested(FieldSortBuilder.java:537)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.build(FieldSortBuilder.java:360)\n\tat org.elasticsearch.search.sort.SortBuilder.buildSort(SortBuilder.java:160)\n\tat org.elasticsearch.search.aggregations.support.AggregationContext$ProductionAggregationContext.buildSort(AggregationContext.java:473)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:643)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:51)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.aggregations.AggregatorFactory.<init>(AggregatorFactory.java:44)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregatorFactory.<init>(ValuesSourceAggregatorFactory.java:31)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregatorFactory.<init>(TermsAggregatorFactory.java:242)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregationBuilder.innerBuild(TermsAggregationBuilder.java:365)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:421)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:31)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.SearchService.parseSource(SearchService.java:1228)\n\tat org.elasticsearch.search.SearchService.createContext(SearchService.java:971)\n\tat org.elasticsearch.search.SearchService.executeQueryPhase(SearchService.java:621)\n\tat org.elasticsearch.search.SearchService.lambda$executeQueryPhase$2(SearchService.java:487)\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply$0(ActionRunnable.java:47)\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:62)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:776)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\n"}],"type":"search_phase_execution_exception","reason":"all shards failed","phase":"query","grouped":true,"failed_shards":[{"shard":0,"index":"my-index","node":"hQuVlgK9SSa9gewozXK1QQ","reason":{"type":"query_shard_exception","reason":"it is mandatory to set the [nested] context on the nested sort field: [users.last.keyword].","index_uuid":"wrg-FT1JRPSAIPWt5qzYTw","index":"my-index","stack_trace":"[my-index/wrg-FT1JRPSAIPWt5qzYTw] org.elasticsearch.index.query.QueryShardException: it is mandatory to set the [nested] context on the nested sort field: [users.last.keyword].\n\tat org.elasticsearch.search.sort.FieldSortBuilder.validateMissingNestedPath(FieldSortBuilder.java:665)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.nested(FieldSortBuilder.java:537)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.build(FieldSortBuilder.java:360)\n\tat org.elasticsearch.search.sort.SortBuilder.buildSort(SortBuilder.java:160)\n\tat org.elasticsearch.search.aggregations.support.AggregationContext$ProductionAggregationContext.buildSort(AggregationContext.java:473)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:643)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:51)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.aggregations.AggregatorFactory.<init>(AggregatorFactory.java:44)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregatorFactory.<init>(ValuesSourceAggregatorFactory.java:31)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregatorFactory.<init>(TermsAggregatorFactory.java:242)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregationBuilder.innerBuild(TermsAggregationBuilder.java:365)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:421)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:31)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.SearchService.parseSource(SearchService.java:1228)\n\tat org.elasticsearch.search.SearchService.createContext(SearchService.java:971)\n\tat org.elasticsearch.search.SearchService.executeQueryPhase(SearchService.java:621)\n\tat org.elasticsearch.search.SearchService.lambda$executeQueryPhase$2(SearchService.java:487)\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply$0(ActionRunnable.java:47)\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:62)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:776)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\n"}}],"stack_trace":"Failed to execute phase [query], all shards failed; shardFailures {[hQuVlgK9SSa9gewozXK1QQ][my-index][0]: org.elasticsearch.transport.RemoteTransportException: [yamlRestTestV7CompatTest-0][127.0.0.1:39761][indices:data/read/search[phase/query]]\nCaused by: [my-index/wrg-FT1JRPSAIPWt5qzYTw] org.elasticsearch.index.query.QueryShardException: it is mandatory to set the [nested] context on the nested sort field: [users.last.keyword].\n\tat org.elasticsearch.search.sort.FieldSortBuilder.validateMissingNestedPath(FieldSortBuilder.java:665)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.nested(FieldSortBuilder.java:537)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.build(FieldSortBuilder.java:360)\n\tat org.elasticsearch.search.sort.SortBuilder.buildSort(SortBuilder.java:160)\n\tat org.elasticsearch.search.aggregations.support.AggregationContext$ProductionAggregationContext.buildSort(AggregationContext.java:473)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:643)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:51)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.aggregations.AggregatorFactory.<init>(AggregatorFactory.java:44)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregatorFactory.<init>(ValuesSourceAggregatorFactory.java:31)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregatorFactory.<init>(TermsAggregatorFactory.java:242)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregationBuilder.innerBuild(TermsAggregationBuilder.java:365)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:421)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:31)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.SearchService.parseSource(SearchService.java:1228)\n\tat org.elasticsearch.search.SearchService.createContext(SearchService.java:971)\n\tat org.elasticsearch.search.SearchService.executeQueryPhase(SearchService.java:621)\n\tat org.elasticsearch.search.SearchService.lambda$executeQueryPhase$2(SearchService.java:487)\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply$0(ActionRunnable.java:47)\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:62)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:776)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\n}\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.onPhaseFailure(AbstractSearchAsyncAction.java:716)\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.executeNextPhase(AbstractSearchAsyncAction.java:403)\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.onPhaseDone(AbstractSearchAsyncAction.java:748)\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.onShardFailure(AbstractSearchAsyncAction.java:500)\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction$1.onFailure(AbstractSearchAsyncAction.java:334)\n\tat org.elasticsearch.action.ActionListener$Delegating.onFailure(ActionListener.java:66)\n\tat org.elasticsearch.action.ActionListenerResponseHandler.handleException(ActionListenerResponseHandler.java:48)\n\tat org.elasticsearch.action.search.SearchTransportService$ConnectionCountingHandler.handleException(SearchTransportService.java:651)\n\tat org.elasticsearch.transport.TransportService$4.handleException(TransportService.java:724)\n\tat org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleException(TransportService.java:1349)\n\tat org.elasticsearch.transport.TransportService$DirectResponseChannel.processException(TransportService.java:1458)\n\tat org.elasticsearch.transport.TransportService$DirectResponseChannel.sendResponse(TransportService.java:1432)\n\tat org.elasticsearch.transport.TaskTransportChannel.sendResponse(TaskTransportChannel.java:50)\n\tat org.elasticsearch.action.support.ChannelActionListener.onFailure(ChannelActionListener.java:47)\n\tat org.elasticsearch.action.ActionRunnable.onFailure(ActionRunnable.java:77)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:28)\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:776)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\nCaused by: [my-index/wrg-FT1JRPSAIPWt5qzYTw] org.elasticsearch.index.query.QueryShardException: it is mandatory to set the [nested] context on the nested sort field: [users.last.keyword].\n\tat org.elasticsearch.search.sort.FieldSortBuilder.validateMissingNestedPath(FieldSortBuilder.java:665)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.nested(FieldSortBuilder.java:537)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.build(FieldSortBuilder.java:360)\n\tat org.elasticsearch.search.sort.SortBuilder.buildSort(SortBuilder.java:160)\n\tat org.elasticsearch.search.aggregations.support.AggregationContext$ProductionAggregationContext.buildSort(AggregationContext.java:473)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:643)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:51)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.aggregations.AggregatorFactory.<init>(AggregatorFactory.java:44)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregatorFactory.<init>(ValuesSourceAggregatorFactory.java:31)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregatorFactory.<init>(TermsAggregatorFactory.java:242)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregationBuilder.innerBuild(TermsAggregationBuilder.java:365)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:421)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:31)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.SearchService.parseSource(SearchService.java:1228)\n\tat org.elasticsearch.search.SearchService.createContext(SearchService.java:971)\n\tat org.elasticsearch.search.SearchService.executeQueryPhase(SearchService.java:621)\n\tat org.elasticsearch.search.SearchService.lambda$executeQueryPhase$2(SearchService.java:487)\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply$0(ActionRunnable.java:47)\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:62)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\t... 6 more\n"},"status":400}]

  at __randomizedtesting.SeedInfo.seed([C4F70F390EC7D2D0:4CA330E3A03BBF28]:0)
  at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:489)
  at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:462)
  at jdk.internal.reflect.GeneratedMethodAccessor12.invoke(null:-1)
  at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at java.lang.reflect.Method.invoke(Method.java:568)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:44)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:824)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:475)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:831)
  at java.lang.Thread.run(Thread.java:833)

  Caused by: java.lang.AssertionError: expected [2xx] status code but api [search] returned [400 Bad Request] [{"error":{"root_cause":[{"type":"query_shard_exception","reason":"it is mandatory to set the [nested] context on the nested sort field: [users.last.keyword].","index_uuid":"wrg-FT1JRPSAIPWt5qzYTw","index":"my-index","stack_trace":"[my-index/wrg-FT1JRPSAIPWt5qzYTw] org.elasticsearch.index.query.QueryShardException: it is mandatory to set the [nested] context on the nested sort field: [users.last.keyword].\n\tat org.elasticsearch.search.sort.FieldSortBuilder.validateMissingNestedPath(FieldSortBuilder.java:665)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.nested(FieldSortBuilder.java:537)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.build(FieldSortBuilder.java:360)\n\tat org.elasticsearch.search.sort.SortBuilder.buildSort(SortBuilder.java:160)\n\tat org.elasticsearch.search.aggregations.support.AggregationContext$ProductionAggregationContext.buildSort(AggregationContext.java:473)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:643)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:51)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.aggregations.AggregatorFactory.<init>(AggregatorFactory.java:44)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregatorFactory.<init>(ValuesSourceAggregatorFactory.java:31)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregatorFactory.<init>(TermsAggregatorFactory.java:242)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregationBuilder.innerBuild(TermsAggregationBuilder.java:365)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:421)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:31)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.SearchService.parseSource(SearchService.java:1228)\n\tat org.elasticsearch.search.SearchService.createContext(SearchService.java:971)\n\tat org.elasticsearch.search.SearchService.executeQueryPhase(SearchService.java:621)\n\tat org.elasticsearch.search.SearchService.lambda$executeQueryPhase$2(SearchService.java:487)\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply$0(ActionRunnable.java:47)\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:62)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:776)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\n"}],"type":"search_phase_execution_exception","reason":"all shards failed","phase":"query","grouped":true,"failed_shards":[{"shard":0,"index":"my-index","node":"hQuVlgK9SSa9gewozXK1QQ","reason":{"type":"query_shard_exception","reason":"it is mandatory to set the [nested] context on the nested sort field: [users.last.keyword].","index_uuid":"wrg-FT1JRPSAIPWt5qzYTw","index":"my-index","stack_trace":"[my-index/wrg-FT1JRPSAIPWt5qzYTw] org.elasticsearch.index.query.QueryShardException: it is mandatory to set the [nested] context on the nested sort field: [users.last.keyword].\n\tat org.elasticsearch.search.sort.FieldSortBuilder.validateMissingNestedPath(FieldSortBuilder.java:665)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.nested(FieldSortBuilder.java:537)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.build(FieldSortBuilder.java:360)\n\tat org.elasticsearch.search.sort.SortBuilder.buildSort(SortBuilder.java:160)\n\tat org.elasticsearch.search.aggregations.support.AggregationContext$ProductionAggregationContext.buildSort(AggregationContext.java:473)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:643)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:51)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.aggregations.AggregatorFactory.<init>(AggregatorFactory.java:44)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregatorFactory.<init>(ValuesSourceAggregatorFactory.java:31)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregatorFactory.<init>(TermsAggregatorFactory.java:242)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregationBuilder.innerBuild(TermsAggregationBuilder.java:365)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:421)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:31)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.SearchService.parseSource(SearchService.java:1228)\n\tat org.elasticsearch.search.SearchService.createContext(SearchService.java:971)\n\tat org.elasticsearch.search.SearchService.executeQueryPhase(SearchService.java:621)\n\tat org.elasticsearch.search.SearchService.lambda$executeQueryPhase$2(SearchService.java:487)\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply$0(ActionRunnable.java:47)\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:62)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:776)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\n"}}],"stack_trace":"Failed to execute phase [query], all shards failed; shardFailures {[hQuVlgK9SSa9gewozXK1QQ][my-index][0]: org.elasticsearch.transport.RemoteTransportException: [yamlRestTestV7CompatTest-0][127.0.0.1:39761][indices:data/read/search[phase/query]]\nCaused by: [my-index/wrg-FT1JRPSAIPWt5qzYTw] org.elasticsearch.index.query.QueryShardException: it is mandatory to set the [nested] context on the nested sort field: [users.last.keyword].\n\tat org.elasticsearch.search.sort.FieldSortBuilder.validateMissingNestedPath(FieldSortBuilder.java:665)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.nested(FieldSortBuilder.java:537)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.build(FieldSortBuilder.java:360)\n\tat org.elasticsearch.search.sort.SortBuilder.buildSort(SortBuilder.java:160)\n\tat org.elasticsearch.search.aggregations.support.AggregationContext$ProductionAggregationContext.buildSort(AggregationContext.java:473)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:643)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:51)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.aggregations.AggregatorFactory.<init>(AggregatorFactory.java:44)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregatorFactory.<init>(ValuesSourceAggregatorFactory.java:31)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregatorFactory.<init>(TermsAggregatorFactory.java:242)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregationBuilder.innerBuild(TermsAggregationBuilder.java:365)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:421)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:31)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.SearchService.parseSource(SearchService.java:1228)\n\tat org.elasticsearch.search.SearchService.createContext(SearchService.java:971)\n\tat org.elasticsearch.search.SearchService.executeQueryPhase(SearchService.java:621)\n\tat org.elasticsearch.search.SearchService.lambda$executeQueryPhase$2(SearchService.java:487)\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply$0(ActionRunnable.java:47)\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:62)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:776)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\n}\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.onPhaseFailure(AbstractSearchAsyncAction.java:716)\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.executeNextPhase(AbstractSearchAsyncAction.java:403)\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.onPhaseDone(AbstractSearchAsyncAction.java:748)\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.onShardFailure(AbstractSearchAsyncAction.java:500)\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction$1.onFailure(AbstractSearchAsyncAction.java:334)\n\tat org.elasticsearch.action.ActionListener$Delegating.onFailure(ActionListener.java:66)\n\tat org.elasticsearch.action.ActionListenerResponseHandler.handleException(ActionListenerResponseHandler.java:48)\n\tat org.elasticsearch.action.search.SearchTransportService$ConnectionCountingHandler.handleException(SearchTransportService.java:651)\n\tat org.elasticsearch.transport.TransportService$4.handleException(TransportService.java:724)\n\tat org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleException(TransportService.java:1349)\n\tat org.elasticsearch.transport.TransportService$DirectResponseChannel.processException(TransportService.java:1458)\n\tat org.elasticsearch.transport.TransportService$DirectResponseChannel.sendResponse(TransportService.java:1432)\n\tat org.elasticsearch.transport.TaskTransportChannel.sendResponse(TaskTransportChannel.java:50)\n\tat org.elasticsearch.action.support.ChannelActionListener.onFailure(ChannelActionListener.java:47)\n\tat org.elasticsearch.action.ActionRunnable.onFailure(ActionRunnable.java:77)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:28)\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:776)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:833)\nCaused by: [my-index/wrg-FT1JRPSAIPWt5qzYTw] org.elasticsearch.index.query.QueryShardException: it is mandatory to set the [nested] context on the nested sort field: [users.last.keyword].\n\tat org.elasticsearch.search.sort.FieldSortBuilder.validateMissingNestedPath(FieldSortBuilder.java:665)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.nested(FieldSortBuilder.java:537)\n\tat org.elasticsearch.search.sort.FieldSortBuilder.build(FieldSortBuilder.java:360)\n\tat org.elasticsearch.search.sort.SortBuilder.buildSort(SortBuilder.java:160)\n\tat org.elasticsearch.search.aggregations.support.AggregationContext$ProductionAggregationContext.buildSort(AggregationContext.java:473)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:643)\n\tat org.elasticsearch.search.aggregations.metrics.TopHitsAggregationBuilder.doBuild(TopHitsAggregationBuilder.java:51)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.aggregations.AggregatorFactory.<init>(AggregatorFactory.java:44)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregatorFactory.<init>(ValuesSourceAggregatorFactory.java:31)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregatorFactory.<init>(TermsAggregatorFactory.java:242)\n\tat org.elasticsearch.search.aggregations.bucket.terms.TermsAggregationBuilder.innerBuild(TermsAggregationBuilder.java:365)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:421)\n\tat org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder.doBuild(ValuesSourceAggregationBuilder.java:31)\n\tat org.elasticsearch.search.aggregations.AbstractAggregationBuilder.build(AbstractAggregationBuilder.java:129)\n\tat org.elasticsearch.search.aggregations.AggregatorFactories$Builder.build(AggregatorFactories.java:403)\n\tat org.elasticsearch.search.SearchService.parseSource(SearchService.java:1228)\n\tat org.elasticsearch.search.SearchService.createContext(SearchService.java:971)\n\tat org.elasticsearch.search.SearchService.executeQueryPhase(SearchService.java:621)\n\tat org.elasticsearch.search.SearchService.lambda$executeQueryPhase$2(SearchService.java:487)\n\tat org.elasticsearch.action.ActionRunnable.lambda$supply$0(ActionRunnable.java:47)\n\tat org.elasticsearch.action.ActionRunnable$2.doRun(ActionRunnable.java:62)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\t... 6 more\n"},"status":400}]

    at org.junit.Assert.fail(Assert.java:88)
    at org.elasticsearch.test.rest.yaml.section.DoSection.execute(DoSection.java:381)
    at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:478)
    at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:462)
    at jdk.internal.reflect.GeneratedMethodAccessor12.invoke(null:-1)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:568)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:44)
    at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
    at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
    at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:824)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:475)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
    at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
    at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
    at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
    at org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:831)
    at java.lang.Thread.run(Thread.java:833)

```

--- a/rest-api-spec/build.gradle
+++ b/rest-api-spec/build.gradle
@@ -76,7 +76,7 @@ tasks.named("yamlRestTestV7CompatTransform").configure { task ->
   task.skipTest("field_caps/30_filter/Field caps with index filter", "behaviour change after #63692 4digits dates are parsed as epoch and in quotes as year")
   task.skipTest("indices.forcemerge/10_basic/Check deprecation warning when incompatible only_expunge_deletes and max_num_segments values are both set", "#44761 bug fix")
   task.skipTest("search/340_type_query/type query", "#47207 type query throws exception in compatible mode")
-  task.skipTest("search.aggregation/200_top_hits_metric/top_hits aggregation with sequence numbers", "#42809 the use nested path and filter sort throws an exception")
+  task.skipTest("search.aggregation/210_top_hits_nested_metric/top_hits aggregation with sequence numbers", "#42809 the use nested path and filter sort throws an exception")
   task.skipTest("search/310_match_bool_prefix/multi_match multiple fields with cutoff_frequency throws exception", "#42654 cutoff_frequency, common terms are not supported. Throwing an exception")
   task.skipTest("search.aggregation/20_terms/string profiler via global ordinals filters implementation", "The profiler results aren't backwards compatible.")
   task.skipTest("search.aggregation/20_terms/string profiler via global ordinals native implementation", "The profiler results aren't backwards compatible.")

